index.js

```diff
@@ -86,8 +88,63 @@ module.exports = postcss.plugin('postcss-bem-linter', function(primaryOptions, s
       });
     }
 
+    function checkGlob(file, globs) {
```

Comment:
Let's use [globby](https://github.com/sindresorhus/globby) instead of implementing that same behavior ourselves.